.travis.yml

```diff
@@ -18,7 +18,7 @@ language:
 
 python:
   - 2.7
-  - 3.3
+  # - 3.3
```

Comment:
Also need to remove these versions from setup.py (https://github.com/frictionlessdata/tableschema-elasticsearch-py/blob/e517800fb680784ee7c18289541177382d6c24de/setup.py#L67-L68).